### PR TITLE
Management command to refresh documents in AWS bucket

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR handles https://github.com/datamade/la-metro-councilmatic/issues/347

### CODE REVIEW

File: councilmatic_core/management/commands/refresh_pic.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                councilmatic_core_billdocument as b_doc
+                JOIN change_bill 
+                ON change_bill.ocd_id=b_doc.bill_id
+                WHERE document_type='V'
+                UNION
+                SELECT DISTINCT url from 
+                councilmatic_core_eventdocument as e_doc 
+                INNER JOIN councilmatic_core_eventagendaitem as e_item
+                ON e_doc.event_id=e_item.event_id
+                WHERE e_item.updated_at >= (NOW() - INTERVAL '1 hour')
```
Comment (hancush) [inline councilmatic_core/management/commands/refresh_pic.py]: can you say more about this filter? a sql comment might be good!

Comment (reginafcompton) [reply councilmatic_core/management/commands/refresh_pic.py]: sure thing!

File: councilmatic_core/management/commands/refresh_pic.py [hunk 2]
```diff
@@ -59,6 +59,8 @@ def _get_urls(self):
                 INNER JOIN councilmatic_core_eventagendaitem as e_item
                 ON e_doc.event_id=e_item.event_id
                 WHERE e_item.updated_at >= (NOW() - INTERVAL '1 hour')
+                /* import_data runs up to four times per hour on the production site. */
+                /* Select items updated in the last hour, as a precaution for getting the most recently created ones. */
```
Comment (hancush) [inline councilmatic_core/management/commands/refresh_pic.py]: more academic questions! is it possible for a document to be updated, without this timestamp being updated?

Comment (hancush) [reply councilmatic_core/management/commands/refresh_pic.py]: (i'm trying to remember back to when this bug first occurred, and i can't remember whether we we'd registered the document had changed.)

Comment (reginafcompton) [reply councilmatic_core/management/commands/refresh_pic.py]: Good question! And, you know what, I think I missed an obvious way to determine if a document was updated – the Event model, which could have an updated date or name or whatever.

**Pull request merged**